### DISCUSSION TITLE
is PDF per class possible?

### DISCUSSION THREAD

**Discussion by Terrox** [farling42/fvtt-pdf-pager#57] [Q&A] [answered]
like with Powered by the Apocalypse games with a specific PDF per playbook, is it possible to set these up by type or is there only 1 default and the rest are per-actor ?

**Comment by farling42** [comment]
This is not currently possible.

In theory, PDF Pager could choose a PDF to use based on a field in the Actor (such as the class), but it doesn't do it currently.

The question arises as to what gets displayed when the Actor is first created (i.e. it doesn't have a class). If a "common" sheet is used before a class is defined, then does the Actor sheet switch to the alternate sheet as soon as a class is added to the Actor, or should it wait until the window is reopened?

**Comment by farling42** [comment, marked as answer]
In case you haven't already found it, once you are using the "PDF Actor Sheet" for your Actor, then you get an additional button ("Custom PDF") in the Actor sheet's title bar which allows you to choose a specific PDF to use for that particular Actor.
It is a manual process, but should only require setting once for each PC Actor.
